grafana/Dockerfile-grafana

```diff
@@ -1,2 +1,7 @@
 FROM grafana/grafana:latest
-COPY --chown=grafana:grafana grafana.db /var/lib/grafana/grafana.db
+USER 0
+RUN apk add sqlite
```

Comment:
Just a general note: Please try to stick with using RHEL or CentOS-based containers.

Because this is more of a developer productivity thing, I won't block this PR on it. But, try to use RHEL/UBI or CentOS when you can. And, at some point in the future when we have some spare time, if you can update this image, we should do that. :slightly_smiling_face:

Reply:
Dang! You're right. I completely missed that. I shall have to try to find a RHEL container or build one.
If you're not going to block the PR (since this is a dev-only tool) then I'll put in another ticket for that work..